### ISSUE TITLE
Katas Bottom Numbers

### ISSUE THREAD

**Issue by DimitriBeman** [tddbin/es6katas.org#21]
Hi guys, es5 Katas it's absolutely great but each time that I refresh the page the Bottom Numbers go away...

I've tried with latest chrome and IE10.

Thanks !!!

**Comment by wolframkriesing** [comment]
yeah, unfortunately the integration of es6katas with tddbin is not ideal yet. Better ideas are welcome, currently I tried to decouple them, which ended in this half backed thingy :(
